python/openvino_tensorflow/__init__.in.py

```diff
@@ -140,9 +144,12 @@ def requested():
 
     def enable():
         openvino_tensorflow_lib.enable()
+        os.environ["CUDA_VISIBLE_DEVICES"] = "-1"
```

Comment:
Shall we capture the current state of this variable in enable() before setting it to -1 and then in disable() we set it back to the same state?